src/sql/workbench/browser/modal/modal.ts

```diff
@@ -56,9 +56,12 @@ export interface IModalDialogStyles {
 }
 
 export type DialogWidth = 'narrow' | 'medium' | 'wide' | number;
+export type DialogStyle = 'Normal' | 'Flyout' | 'Callout';
```

Comment:
@Charles-Gagnon All three types: DialogWidth, DialogStyle and DialogPosition exist in sqlExtHostTypes and in azdata.proposed.d.ts. I added them here to on the advice of another engineer. Where MUST these three types reside, and how do I refer to them further down in my code if they don't reside in modal.ts?

Reply:
You should be able to import them directly from azdata since these are just static values.

But given that we already import DialogWidth in this way then I guess there's no harm in doing it this way too.